stable/mcrouter/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: mcrouter
+home: https://github.com/facebook/mcrouter
+version: 0.1.0
+appVersion: 1.0
+description: Mcrouter is a memcached protocol router for scaling memcached deployments.
+sources:
+- https://github.com/facebook/mcrouter
+- https://github.com/jamescarr/mcrouter/
+maintainers:
+- name: Julien Phalip
```

Comment:
Confusingly this should be your Github ID nowadays. Ill add something in the contributing notes and Helm docs.

Apologies.